src/models/Role.js

```diff
@@ -0,0 +1,21 @@
+export default (sequelize, DataTypes) => {
+  const Role = sequelize.define('role', {
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    parentId: {
```

Comment:
Please what is parentId?

Reply:
Roles can inherit permissions from other roles( Hierarchical role-based access control), parentId referenced the parent role. 
I will be removing this functionality as I have come to the conclusion that it isn't important